packages/apps/auth0-database-scripts/src/create.ts

```diff
@@ -66,29 +63,18 @@ async function create(user: Auth0UserWithPassword) {
     ) {
       throw new ValidationError(user.email, 'Please use a shorter password.');
     } else {
-      console.log('CREATE PATRON IN SIERRA ERRORS');
       throw new Error(createPatronResponse.message);
     }
   }
 
-  // We now need to find the patron we created and eventually update their barcode
-  const findPatronResponse = await sierraClient.getPatronRecordByEmail(
-    user.email
-  );
-  if (findPatronResponse.status !== ResponseStatus.Success) {
-    console.log('FIND PATRON BARCODE - NOT SUCCESSFUL');
-    throw new Error(findPatronResponse.message);
-  }
-  const { recordNumber } = findPatronResponse.result;
```

Comment:
The creation response contains the new patron's record number inside their URI, so we can remove this request